# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libetpan/0002-fix-date_time_new.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 5ba50e1a12d0ac27d2158d0c9546f4adde888568 Mon Sep 17 00:00:00 2001
From: Nadeem Shehzad <Nadeem.Shehzad@securepoint.de>
Date: Tue, 25 Jan 2011 12:09:38 +0100
Subject: [PATCH 2/3] fix: date_time_new by correcting seconds assingment to dt_sec rather than dt_day.

---
 src/low-level/imap/mailimap_types.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/low-level/imap/mailimap_types.c b/src/low-level/imap/mailimap_types.c
index aee05de..322846b 100644
--- a/src/low-level/imap/mailimap_types.c
+++ b/src/low-level/imap/mailimap_types.c
@@ -836,7 +836,7 @@ mailimap_date_time_new(int dt_day, int dt_month, int dt_year, int dt_hour,
   date_time->dt_year = dt_year;
   date_time->dt_hour = dt_hour;
   date_time->dt_min = dt_min;
-  date_time->dt_day = dt_sec;
+  date_time->dt_sec = dt_sec;
   date_time->dt_zone = dt_zone;
 
   return date_time;
-- 
1.6.3.3

